test/test_helper.rb

```diff
@@ -49,3 +49,6 @@ def in_file(i)
   File.expand_path(File.dirname(__FILE__) + '/fixtures/files/' + i)
 end
 
+def travis?
+  ENV['TRAVIS']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.